tools/genesis/genesis.go

```diff
@@ -104,14 +104,9 @@ func generateAllocations(
 			continue
 		}
 
-		// Computation of the offer value as a key to the map of DepositOffers
-		YearToSeconds := float64(365 * 24 * 60 * 60)
-		offerValueMinDuration := uint64((al.UnbondingStart + al.UnbondingPeriod) * YearToSeconds)
-		offerValueUnlockPeriodDuration := uint64(al.UnbondingPeriod * YearToSeconds)
-		offerValueIndex := strconv.FormatUint(offerValueMinDuration, 10) + "_" + strconv.FormatUint(offerValueUnlockPeriodDuration, 10) + "_" + strconv.FormatInt(int64(al.RewardPercent), 10)
-
+		offer, hasOffer := offersMap[al.OfferID]
 		directAmount := uint64(0)
-		if offerValueMinDuration == 0 && offerValueUnlockPeriodDuration == 0 {
+		if !hasOffer {
```

Comment:
That should only happen if the depositOffer ID is actually empty - if it's not found we must throw an error as it's most likely an error in the Excel file